views.py

```diff
@@ -505,7 +508,9 @@ def resource_edit(resource_type, resource_id):
 
     for attribute in editable_attributes:
         attribute_value = request.form.get(attribute["name"])
-        if (
+        if attribute["name"] == "password":
```

Comment:
we have a configurable column name in `admin_view.py` within `admin_configs` variable for the identifier secret. Let's use that here. So essentially it will look something like this:
```py
if attribute["name"] == admin_configs['user']['secret']:
```